coins/bitcoin/worker/src/main/scala/co/ledger/lama/bitcoin/worker/App.scala

```diff
@@ -1,28 +1,58 @@
 package co.ledger.lama.bitcoin.worker
 
-import cats.effect.{ExitCode, IO, IOApp}
+import java.util.concurrent.Executors
+
+import cats.effect.{Blocker, ExitCode, IO, IOApp, Resource}
+import co.ledger.lama.bitcoin.worker.config.Config
 import co.ledger.lama.bitcoin.worker.services._
+import dev.profunktor.fs2rabbit.interpreter.RabbitClient
+import org.http4s.client.blaze.BlazeClientBuilder
 import pureconfig.ConfigSource
 
+import scala.concurrent.ExecutionContext
+
 object App extends IOApp {
 
   def run(args: List[String]): IO[ExitCode] = {
     val conf = ConfigSource.default.loadOrThrow[Config]
 
-    val rabbitService = new RabbitService(
-      conf.rabbit,
-      conf.queueName(conf.workerExchangeName),
-      conf.lamaExchangeName,
-      conf.routingKey
-    )
+    val resources = for {
+      rabbitClient <-
+        Resource
+          .make(IO(Executors.newCachedThreadPool()))(es => IO(es.shutdown()))
+          .map(Blocker.liftExecutorService)
+          .evalMap(RabbitClient[IO](conf.rabbit, _))
+
+      httpClient <- BlazeClientBuilder[IO](
+        ExecutionContext.fromExecutorService(Executors.newCachedThreadPool)
+      ).resource
+
```

Comment:
Also this.
it is also better to name those pools to have better error logs.
```suggestion
      httpClient <- Clients.http4s
```